### PR TITLE
Branch zhou for 1.0b

### LINKED ISSUE #24: Add parameters to biometrics
interpret user's input about their biometrics

### PR DESCRIPTION
**Pull request by maanyos**
fixes #24 
add Junit tests for Parser
updated text ui tests
add invalid command handling
change Parser separator to '/'

### CODE REVIEW

File: src/main/java/seedu/duke/command/SetCommand.java [hunk 1]
```diff
@@ -25,31 +28,70 @@ public SetCommand(String arguments) {
...
+            if (argumentList.length < 5) {
+                throw new IllegalValueException("Insufficient parameters");
             }
-            biometrics.setBiometrics(age, argumentList[2], height, weight, fatPercentage);
-            ui.output(BIOMETRICS_SET);
+            int age = extractAge(argumentList[1].trim());
+            String gender = extractGender(argumentList[2].trim());
+            int height = extractHeight(argumentList[3].trim());
+            int weight = extractWeight(argumentList[4].trim());
+            int fatPercentage = extractFatPercentage(argumentList[5].trim());
```
Comment (gaoyunfan) [inline src/main/java/seedu/duke/command/SetCommand.java:48]: fat is at index 5 so argumentList should have length of 6?

Comment (maanyos) [reply src/main/java/seedu/duke/command/SetCommand.java:48]: oh yes good spot

Comment (maanyos) [reply src/main/java/seedu/duke/command/SetCommand.java]: fixed

File: src/test/java/seedu/duke/ParserTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Test
+    void parse_RandomInput_InvalidCommand() {
+        String userInput = "ding6 Dong66";
+        Command command = Parser.parse(userInput);
+        assertTrue(command instanceof InvalidCommand);
+    }
+
+    @Test
+    void getArgumentList_FiveSeparators_SixArguments() {
```
[the remark this answers is not in the document]
Comment (maanyos) [reply src/test/java/seedu/duke/ParserTest.java:28]: ok will change

File: src/main/java/seedu/duke/command/SetCommand.java [hunk 3]
```diff
@@ -25,31 +28,70 @@ public SetCommand(String arguments) {
...
+        } catch (IllegalValueException e) {
+            ui.output(e.getMessage());
+            logger.log(Level.WARNING, e.getMessage());
+        }
     }
 
-    private void setBiometrics(String[] argumentList) {
+    private void setBiometrics(String[] argumentList) throws IllegalValueException {
         try {
-            if (argumentList.length < 6) {
```
[the remark this answers is not in the document]
Comment (maanyos) [reply src/main/java/seedu/duke/command/SetCommand.java:33]: extra parameters get ignored anyways

**Pull request closed without merging**